CIP-ShawnMcMurdo-NonCentralizingRankings/CIP-ShawnMcMurdo-NonCentralizingRankings.md

```diff
@@ -0,0 +1,105 @@
+---
+CIP: ?
+Title: Non-Centralizing Rankings
+Author: Shawn McMurdo <shawn_mcmurdo@yahoo.com>
+Discussions-To: 
+Comments-Summary: 
+Comments-URI: 
+Status: Draft
+Type: Standards
```

Comment:
I think this should be an "Informational" since this doesn't affect the protocol itself